app/javascript/stylesheets/navbar.scss

```diff
@@ -102,14 +102,14 @@ nav {
   }
 
   nav {
-    ul, form input {
+    ul, input {
       width: 100%;
     }
 
     ul {
       display: none;
 
-      form input, a {
+      input, a {
         padding: 20px 0;
```

Comment:
The reason I did for form input instead of just input for all of these is because the checkbox will be styled by everything here since it is also an input.

Since it has display: none it doesn't really matter.  However I'm not a fan of applying styling to elements which are redundant or aren't needed.